### DISCUSSION TITLE
Task04 - Problem 5

### DISCUSSION THREAD

**Discussion by SMRazaRizvi96** [chulminy/CIVE497-CIVE700#161] [Q&A] [answered]
For Task04 - Problem 5, should the code create a pop-up window for the user to interactively select the 4 coordinates of the whiteboard, and once recorded, then it should perform the projective transformation?

Or, we can hard-code the corner coordinates in our code?

**Comment by chulminy** [comment, marked as answer]
See this: 

![image](https://github.com/chulminy/CIVE497-CIVE700/assets/3227103/dcc9c9d5-7786-4764-9629-ff9ad2302a60)

**Reply by SMRazaRizvi96** [reply]
Ok. Thanks.
